src/UpcomingBooking/UpcomingBooking.js

```diff
@@ -296,24 +301,34 @@ class UpcomingBooking extends React.Component<Props, State> {
     return (
       <div className="UpcomingBooking">
         <div className="Screen-title">
+          <div className="Screen-title-group">
+            <div className="booking-id">
+              {booking.databaseId && (
+                <Typography type="secondary">
+                  Upcoming booking # {formatBookingId(booking.databaseId)}
```

Comment:
This should read `Upcoming trip` as per design.

Reply:
:man_facepalming: